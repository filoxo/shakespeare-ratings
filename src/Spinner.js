import React from 'react'
import './Spinner.css'

const Spinner = () => (
  <span className="spinner" aria-busy="true" role="alert">
    <span className="hideVisually">Loading...</span>
  </span>
)

export default Spinner
