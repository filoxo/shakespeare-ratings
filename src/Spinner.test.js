import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import Spinner from './Spinner'

describe('<Spinner/>', () => {
  it('renders without errors', () => {
    const wrapper = shallow(<Spinner />)
    expect(wrapper).toBeTruthy()
    const tree = renderer.create(<Spinner />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
