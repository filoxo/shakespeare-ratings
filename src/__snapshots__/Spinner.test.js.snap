// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner/> renders without errors 1`] = `
<span
  aria-busy="true"
  className="spinner"
  role="alert"
>
  <span
    className="hideVisually"
  >
    Loading...
  </span>
</span>
`;
